modules/installation-aws-config-yaml-customizations.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/ipi/installing-aws-customizations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="installation-aws-config-yaml-customizations_{context}"]
+= Sample customized install-config.yaml file for AWS
+
+You can customize the installation configuration file (`install-config.yaml`) to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+For a full list and description of all installation configuration parameters, see the page titled _Installation configuration parameters_.
+====
+
+[source,yaml]
```

Comment:
Would it help to add a title to the YAML block?

Reply:
Definitely